GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,6 +474,24 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
+octopus = ItemKind
+  { isymbol  = 'o'
+  , iname    = "Giant octopus"
+  , ifreq    = [ ("animal", 100), ("mobile", 1), ("mobile animal", 100) ]
+  , iflavour = zipPlain [BrMagenta]
+  , icount   = 1
+  , irarity  = [(1, 5)]
+  , iverbHit = "thud"
+  , iweight  = 15000
```

Comment:
They are said to weight up to 70kg, but probably most of the mass is in the tentacles, so 15kg for the trunk makes sense. OTOH, if you wish, you can exaggerate a bit and make our own octopus a sight to behold. :) OTOH, if you go for an exceptionally large specimen, you may want to bump HP, given that it doesn't have armored skin and so is rather fragile.

Reply:
The record is 71kg, so I'm going to make it 72kg ;)

Reply:
Perfect. I guess the lower gravity servers them well. :)